client/actions/book_actions.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+import { ADD_BOOK, GET_ALL_BOOKS, GET_RENTED_BOOKS } from './types';
+
+const API_URL = 'http://localhost:8000/api/v1/books',
+  USER_API_URL = 'http://localhost:8000/api/v1/users';
+
+export function addNewBook(bookDetails) {
+  return dispatch => axios.post(`${API_URL}`, bookDetails)
```

Comment:
Since you're only passing the variable API_URL, why are you using a string literal? You should just pass in the variable name alone without `${}`

Reply:
Thanks for that, I will surely fix it up.